esp/src/src-react/components/Sasha.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+
```

Comment:
This component should probably be using the FluentUI components for any HTML it returns.